### PR TITLE
Adding translations for parts_of_story.md

### PR DESCRIPTION
**Pull request by rachnakumar**
Adding es, id, ja, ko, pt_BR, zh_CN translations for parts_of_story.md

### CODE REVIEW

File: content/docs/getting_started/visual_story/parts_of_story@es.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+---
+$title: Partes de las historias de AMP
+$order: 1
+---
```
[the remark this answers is not in the document]
Comment (rachnakumar) [reply content/docs/getting_started/visual_story/parts_of_story@es.md]: Updated all the translation files with the front matter changes.

**Comment by CrystalFaith** [comment]
Adding @techhtml and @captain-lee to look at the .ko translation. Front matter looks good, thanks!

**Comment by techhtml** [comment]
I think an original translation is better than the new translation.
how about remove new ko translations?

File: content/docs/getting_started/visual_story/parts_of_story@ko.md [hunk 2]
```diff
@@ -1,24 +1,18 @@
 ---
-$title: AMP 스토리 내부구조 이해하기
+$title: AMP 스토리의 구성요소 이해하기
 ---
 
-AMP 스토리는 화면을 꽉채우는 시각효과를 통해 이야기를 전달합니다. 이미지,
-비디오, 그래픽 및 오디오 등의 정보와 함께 이야기를 전달하는데요. 읽기 간편하고
-시각적으로 풍부한 콘텐츠를 좋아하는 사용자에게 안성맞춤입니다.
+AMP 스토리는 전체 화면에서 이미지, 동영상, 그래픽, 오디오 등으로 정보를 전달하는 시각적 스토리텔링 도구로, 바이트 단위의 공간만 사용하면서 시각적으로 풍부한 콘텐츠를 원하는 사용자에게 이상적입니다.
```
Comment (captain-lee) [inline content/docs/getting_started/visual_story/parts_of_story@ko.md]: Incorrect translation: "바이트 단위의 공간만 사용하면서" (using byte size space). The corresponding English text is "bite-sized" which means short or easy-to read, not computer byte-sized.
Proposed: "읽기 간편하고"

**Comment by CrystalFaith** [comment]
@captain-lee and @techhtml - I'm okay with keeping the old doc if its better. Is there anything in the old doc we could update with this PR? Thanks!

File: content/docs/getting_started/visual_story/parts_of_story@ko.md [hunk 3]
```diff
@@ -1,24 +1,18 @@
...
-하나의 AMP 스토리가 됩니다. 또 페이지는 여러 **레이어(layer)**들이 모인것이고
-레이어안에는 HTML 또는 AMP **요소(element)**들이 들어있습니다.
+AMP 스토리의 기본 구성요소는 개별 **페이지**입니다. 이러한 페이지는 기본 HTML 및 AMP **요소**가 포함된 개별 **레이어**로 구성됩니다.
 
 {{ image('/static/img/docs/tutorials/amp_story/story_parts.png', 1047, 452, align='center ninety') }}
 
-AMP 스토리를 구성하는 부품을 기술용어로는 AMP 컴포넌트라고 표현하는데 AMP
-스토리는 `amp-story`, 페이지는 `amp-story-page` 그리고 레이어는
-`amp-story-grid-layer` 컴포넌트로 표현합니다.
+이러한 각 구성요소는 AMP 구성요소로 변환되며, 여기서 스토리는 `amp-story`로, 페이지는 `amp-story-page`로, 그리고 레이어는 'amp-story-grid-layer`로 표시됩니다.
```
[the remark this answers is not in the document]
Comment (captain-lee) [reply content/docs/getting_started/visual_story/parts_of_story@ko.md]: My previous message doesn't display the character I'm referring: \' -> \`, for example,
\`amp-story-grid-layer`

File: content/docs/getting_started/visual_story/parts_of_story@ko.md [hunk 4]
```diff
@@ -1,24 +1,18 @@
...
 
 {{ image('/static/img/docs/amp-story-tag-hierarchy.png', 557, 355, align='center seventyfive' ) }}
 
-자, 그럼 `amp-story` 컨테이너를 가지고 본격적으로 이야기를 시작해보겠습니다.
+`amp-story` 컨테이너를 사용하여 스토리 만들기를 시작하겠습니다.
```
[the remark this answers is not in the document]
Comment (captain-lee) [reply content/docs/getting_started/visual_story/parts_of_story@ko.md]: +1 for @techhtml's comment with one modification: "만들어봅시다." -> "만들어 보겠습니다." to be consistent with other sentences, for example,

\`amp-story` 컨테이너를 사용하여 스토리를 만들어 보겠습니다.

File: content/docs/getting_started/visual_story/parts_of_story@ko.md (hunk 2 above)
Comment (captain-lee) [reply content/docs/getting_started/visual_story/parts_of_story@ko.md]: So the updated sentence should be
"AMP 스토리는 전체 화면에서 이미지, 동영상, 그래픽, 오디오 등으로 정보를 전달하는 시각적 스토리텔링 도구로, 읽기 간편하고 시각적으로 풍부한 콘텐츠를 원하는 사용자에게 이상적입니다."

**Pull request merged**